### ISSUE TITLE
Video Autoplays the Up next video no matter what.

### ISSUE THREAD

**Issue by RohithKumarSp** [YePpHa/YouTubeCenter#2065]
i have turned off auto play, but it plays the next video no matter what you do.
also sometimes i have to refresh my screen 5 times to play a video, as the video pauses itself as soon as i press play.

**Comment by Eisys** [comment]
Might I recommend and alternative to YTC. It might fit your needs.
https://github.com/ParticleCore/Particle/

**Comment by RohithKumarSp** [comment]
yeah, i need some i dont like it much, i have so many features i need from YTC that particle core just doesn't offer me the options.

**Comment by Eisys** [comment]
Ok. I myself got lucky with YT+(Particle) having all the features I used with YTC.
It seems that YTC is dying since it very rarely gets updated and new problems emerge almost daily. So you might want to start looking for an alternative.
